android/fastlane/Fastfile

```diff
@@ -91,11 +91,12 @@ platform :android do
 
   desc "Upload to Beta by Fabric"
   lane :deploy_fabric do |options|
+    File.write("./notes.txt", options[:notes])
```

Comment:
https://github.com/fastlane/fastlane/issues/6325#issuecomment-318026228

Reply:
Ahh good finding. This one was really annoying :+1:

Reply:
Don’t know why it never happened before merging Crashlytics. Build is still flaky because it fetches android tools every time

Reply:
It's the Java dependencies, right? Downloading from `jcenter` seems flaky for some reason. We might be able to add these libraries to the Circle CI cache. This should make the build faster. And maybe less flaky.